src/components/Main/Main.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import TaskList from "../TaskList/TaskList";
+import CreateTask from "../CreateTask/CreateTask";
+import UpdateTask from "../UpdateTask/UpdateTask";
+import DeleteTask from "../DeleteTask/DeleteTask";
+//import Task from "../Task/Task";
+//import { useNavigate } from "react-router-dom";
+import { deleteTask, createTask, updateTask } from "../../utils/api";
+
+const Main = () => {
+  const [tasks, setTasks] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [selectedTask, setSelectedTask] = useState(null);
+  const [activeModal, setActiveModal] = useState("");
```

Comment:
* Please, don't create a lot of `activeModal, setActiveModal` in different components. You have it in `App` and you need to pass `handleCloseModal` to the props of any component that needs it. Or you can create `AppContext` for that